test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -1025,6 +1027,40 @@ def test_qconfig_dict_object_type_module(self):
         self.assertTrue(isinstance(mq[1], nn.Hardswish))
         self.assertTrue(isinstance(mq[2], nnq.Conv2d))
 
+    def test_qconfig_dict_object_type_function(self):
+        """
+        Verifies that the 'object_type' option of qconfig_dict works
+        on function types.
+        """
+        class M(nn.Module):
+            def forward(self, x):
+                x = x + x
```

Comment:
actually, isn't this operator.add?

Reply:
DBR quant will see this after all the PyTorch transformations, as `torch.Tensor.add`, so the knowledge that the user code had `operator.add` is not something that is visible to DBR quant.  I was planning to add test coverage for all the types of add in a future PR, this PR is just to test the path for qconfig_dict with a single thing.